src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1255,6 +1255,8 @@ private AuthenticationResult acquireTokenLocalCall(final CallbackHandler callbac
                         ADALError.DEVELOPER_AUTHORITY_IS_NOT_VALID_INSTANCE));
                 return null;
             }
+        } else {
+            Logger.i(TAG , "Authority validation is turned off:" + authorityUrl.getHost(), "");
```

Comment:
authority host is not PII, it's safe to log it in the message. The intention is to only move the PII into additional message.